apps/storage/routes/attachment/[orgSlug]/[attachmentId]/[filename].ts

```diff
@@ -0,0 +1,78 @@
+import { getSignedUrl } from '@aws-sdk/s3-request-presigner';
+import { GetObjectCommand } from '@aws-sdk/client-s3';
+import { s3Client } from '../../../../utils/s3';
+import { db } from '@uninbox/database';
+import { and, eq } from '@uninbox/database/orm';
+import { convoAttachments, orgMembers, orgs } from '@uninbox/database/schema';
+
+/**
+ * provides a proxy to attachments after verifying the user has access to the attachment
+ * path: storageServer/attachment/[orgSlug]/[attachmentId]/[filename.ext]
+ */
+export default eventHandler({
+  async handler(event) {
+    const orgSlug = getRouterParam(event, 'orgSlug');
+    const attachmentPublicId = getRouterParam(event, 'attachmentId');
+    const filename = getRouterParam(event, 'filename');
```

Comment:
I think this is quite scary because If we ever move this file this wont error. We should add some checks if they are defined

Reply:
move the file?
can you clarify

we only store the attachmentId and filename in the db, not the full path.
thats generated dynamically on the FE